br/pkg/lightning/common/util.go

```diff
@@ -119,8 +119,8 @@ outside:
 		switch {
 		case err == nil:
 			return nil
-		// do not retry NotFound error
-		case errors.IsNotFound(err):
+		// do not retry NotFound error or ErrNoRows
+		case errors.IsNotFound(err), errors.ErrorEqual(err, sql.ErrNoRows):
 			break outside
 		case utils.IsRetryableError(err):
```

Comment:
shouldn't the `ErrNoRows` error be caught in `IsRetryableError` and thus falling into the `default:` branch?

Reply:
yes, this change is useless, I will remove it.